dmutils/email.py

```diff
@@ -1,17 +1,18 @@
 import base64
 import hashlib
-import six
-import struct
 import json
+import struct
+from collections import OrderedDict
 from datetime import datetime
 
+import itsdangerous
```

Comment:
I've taken to working off of http://google.github.io/styleguide/pyguide.html?showone=Imports_formatting#Imports_formatting

Std lib
3rd party
Our libraries (I usually an extra section 'our' libraries here)
then local imports

All in depth then alphabetical order

```
import base64
import hashlib
import json
import struct
from collections import OrderedDict
from datetime import datetime

import itsdangerous
import six
from cryptography import fernet
from flask import current_app
from flask._compat import string_types
from mandrill import Mandrill, Error

from notifications_python_client import NotificationsAPIClient
from notifications_python_client.errors import HTTPError

from .formats import DATETIME_FORMAT
```

I'm easy if there's another convention, tbh I do this predominantly because my editor does it for me :)